erpnext/stock/get_item_details.py

```diff
@@ -220,17 +220,18 @@ def get_basic_details(args, item):
 	if item.variant_of:
 		item.update_template_tables()
 
-	from frappe.defaults import get_user_default_as_list
-	user_default_warehouse_list = get_user_default_as_list('Warehouse')
```

Comment:
I think, this is for setting values based on "User Permission". Should we remove it?

Reply:
User permissions works in the child table, why we need code for that